doc/terms/index.md

```diff
@@ -105,6 +105,174 @@ And this filter validates the second document:
 }
 ```
 
+## match
+
+Test if properties of the filter is matching with the properties of the document.
+
+If A and B both are Scalars, checks that they're equal.
```

Comment:
Can you use the term `filter` for A and `subject` for B? (it's a naming proposal for more clarity and I'm open to other suggestions)